app/navigation/ItemsNavigator.scala

```diff
@@ -53,4 +58,31 @@ class ItemsNavigator @Inject() extends BaseNavigator {
     case ReviewMode =>
       reviewRouteMap(page)(userAnswers)
   }
+
+
+  private def alcoholStrengthRouting(idx: Index, userAnswers: UserAnswers): Call =
+    (userAnswers.get(ItemExciseProductCodePage(idx)), userAnswers.get(ItemAlcoholStrengthPage(idx))) match {
+      case (Some(epc), Some(abv)) =>
+        GoodsTypeModel(epc) match {
+          case Beer =>
+            if (Seq(NorthernIrelandRegisteredConsignor, NorthernIrelandWarehouseKeeper).contains(UserType(userAnswers.ern))) {
+              //TODO: Redirect to CAM-ITM07
+              testOnly.controllers.routes.UnderConstructionController.onPageLoad()
+            } else if (Seq(GreatBritainRegisteredConsignor, GreatBritainWarehouseKeeper).contains(UserType(userAnswers.ern)) && abv < 8.5) {
+              //TODO: Redirect to CAM-ITM41
+              testOnly.controllers.routes.UnderConstructionController.onPageLoad()
+            } else {
+              //TODO: Redirect to CAM-ITM19
```

Comment:
Perhaps a question for a BA, should it go CAM-ITM19 when the ABV is 8.5?

Reply:
The ticket seems to suggest:
 less than 8.5 go to ITM41
 more than 8.5 go to ITM19

However, there doesn't seem to be a scenario when it's equal to 8.5

Reply:
Yeah, it's a good question. I made an assumption but will check.